packages/mdc-dialog/util.js

```diff
@@ -23,9 +23,37 @@
 
 import createFocusTrap from 'focus-trap';
 
-export function createFocusTrapInstance(surfaceEl, acceptButtonEl, focusTrapFactory = createFocusTrap) {
-  return focusTrapFactory(surfaceEl, {
-    initialFocus: acceptButtonEl,
-    clickOutsideDeactivates: true,
-  });
+class MDCDialogUtil {
+  /**
+   * @param {!Element} surfaceEl
+   * @param {?Element=} initialFocusEl
+   * @param {function(!Element, !FocusTrapCreateOptions): !FocusTrapInstance} focusTrapFactory
+   * @return {!FocusTrapInstance}
+   */
+  createFocusTrapInstance(surfaceEl, initialFocusEl = null, focusTrapFactory = createFocusTrap) {
+    return focusTrapFactory(surfaceEl, {
+      initialFocus: initialFocusEl,
+      clickOutsideDeactivates: false,
+    });
+  }
+
+  /**
+   * @param {!Element} el
+   * @return {boolean}
+   */
+  isScrollable(el) {
+    return el.scrollHeight > el.offsetHeight;
+  }
+
+  /**
+   * @param {!Array<!Element>|!NodeList} els
+   * @return {boolean}
+   */
+  areTopsMisaligned(els) {
```

Comment:
It occurs to me that this function is the inverse of what it's used to check in the adapter implementation... should we rename/flip this?

Reply:
Wait, nevermind, misaligned is true when stacked is also true. Dunno why I thought otherwise for a minute.